CC Dashboard: Formatting issues on android tablet

darrenmason opened the pull request:
regarding: https://www.pivotaltracker.com/story/show/127123201

updated button spacing and icon position

![screenshot from 2016-09-02 11 42 12](https://cloud.githubusercontent.com/assets/954569/18209535/28b40946-7102-11e6-991e-857f2157fe06.png)

nathanstitt commented:
Hm, that does fix the "Detailed Scores" button, but the wording beside the icons still doesn't look very good.  Is there anyway to keep the icon from butting up against the text, and fixing the line-height of the labels when they wrap?  

It also seems like the icon should appear after the text since that's how all the other tooltip info icons are displayed elsewhere.

darrenmason commented:
I moved the icon after the text and changed the text to make it more readable and I shortened it slightly, let me know if that works.  (The word "Performance" is implied and can be read in the tooltip)

nathanstitt commented:
Looks good @darrenmason, thanks!
